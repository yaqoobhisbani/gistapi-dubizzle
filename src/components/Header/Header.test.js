import React from "react";
import { render } from "@testing-library/react";
import Header from "./Header";

describe("Header", () => {
  test("renders stop icon", () => {
    const { getByTestId } = render(<Header />);
    const iconElement = getByTestId("icon");
    expect(iconElement).toBeInTheDocument();
  });

  test("matches snapshot", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
