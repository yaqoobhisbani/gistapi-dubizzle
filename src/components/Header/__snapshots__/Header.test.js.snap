// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<div>
  <div
    class="sc-hLseeU dMPsfm"
  >
    <span
      class="mega-octicon octicon-mark-github"
      data-testid="icon"
    />
    <div
      class="sc-beqWaB ihoHfr"
    >
      <div
        class="sc-gueYoa eOwMm"
      >
        <span
          class="octicon octicon-search"
        />
        <input
          class="sc-dmqHEX bqMem"
          data-testid="input"
          placeholder="Search Gists for the username"
          value=""
        />
      </div>
    </div>
  </div>
</div>
`;
